lesson25/css/style.css

```diff
@@ -0,0 +1,175 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400);
+
+body{
+    font-family: 'Open Sans', sans-serif;
+    background-color: #E9EFC0;
+    color: #5e5e5e;
+}
+
+body.fixed{
+    overflow-y: hidden;
+}
+
+.title{
+    text-align: center;
+    font-size: 24px;
+    margin-left: -10px;
+}
+
+.box{
+    background: #fafafa;
+    margin: 100px auto;
+    padding: 40px 30px;
+    max-width: 370px;
+}
+
+.form__inner{
+    margin-top: 30px;
+}
+
+.form__item input{
+    width: 100%;
+    height: 25px;
+    background: #fafafa;
+    color: #666;
+    border-bottom: 2px solid #e9e9e9;
+    font-family: 'Open Sans', sans-serif;
+    font-size: 16px;
+    transition: border-color 0.3s ease-in-out;
+}
+
+.form__item input:focus{
+    border-bottom: 2px solid #c0c0c0;
+}
+
+.form__item + .form__item{
+    margin-top: 15px;
+}
+
+.form__notion{
+    font-size: 12px;
+}
+
+.form__required{
+    color: white;
+    background-color: #ee3e52;
+    padding: 2px 5px 2px 4px;
+    border-radius: 6px;
+    font-size: 11px;
+    margin-left: 6px;
+}
+
+.form__check{
+    margin-top: 30px;
+}
+
+.form__check-link{
+    cursor: pointer;
+    border-bottom: 1px solid #5e5e5e;
+    transition: border-color 0.5s ease-in-out;
+}
+
+.form__check-link:hover{
+    border-bottom: transparent;
+}
+
+.form__attention{
+    display: block;
+    font-size: 13px;
+    padding-left: 15px;
+}
+
+.form__button{
+    margin-top: 10px;
+}
+
+.form__button input{
+    width: 100%;
+    height: 50px;
+    text-align: center;
+    letter-spacing: 0.05em;
+    margin-top: 15px;
+    color: #fff;
+    background-color: #ccc;
+    font-family: 'Open Sans', sans-serif;
+    font-size: 16px;
+    transition: opacity 0.3s ease-in-out;
+}
+
+.form__button input:not([disabled]) {
+    background-color: #83BD75;
+}
+
```

Comment:
IMO
全ての項目がvalidの場合、ブラウザのどこかをクリックしないと
checkFormValidityToEnableSubmitButtonが実行されない為バリデーションが走らないと思います。
つまりsubmitボタンを押す前に、どこかしらをクリックしなくてはいけないと思います。
![b5885fdee9ed0fe5561f9e152cc73ed9](https://user-images.githubusercontent.com/76232904/179325023-35754396-03f6-42f0-9fea-ea47a93590e7.gif)

submitボタンに[pointer-events](https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events)を指定することで解決することができます。
```suggestion
.form__button input{
    width: 100%;
    height: 50px;
    text-align: center;
    letter-spacing: 0.05em;
    margin-top: 15px;
    color: #fff;
    background-color: #ccc;
    font-family: 'Open Sans', sans-serif;
    font-size: 16px;
    transition: opacity 0.3s ease-in-out;
    pointer-events: none;
}

.form__button input:not([disabled]) {
    background-color: #83BD75;
    pointer-events: auto;
}

```

Reply:
Thank you for the solution as well.
I got to know 'pointer-events'.
So we can control them with CSS!
I fixed it!
[05d48bf](https://github.com/haru-programming/JavaScript-lessons/pull/29/commits/05d48bf9de5423738790cb2f1890fcfa0ba09c05)